programs/programs/il/pe/spm.py

```
@@ -48,3 +48,21 @@ class IlTanf(Tanf):
+
+class IlLifeline(Lifeline):
+    pe_name = "lifeline"
+    pe_inputs = [
+        dependency.spm.BroadbandCostDependency,
+        *dependency.irs_gross_income,
+        dependency.spm.Snap,
+    ]
+    pe_outputs = [dependency.spm.Lifeline]
+
+    def postprocess(self, inputs, outputs):
+        lifeline = outputs[dependency.spm.Lifeline]
```

Ruff (0.13.1)

Local variable `lifeline` is assigned to but never used

Remove assignment to unused variable `lifeline`

(F841)